pkg/new-ui/v1beta1/frontend/src/app/pages/experiment-details/experiment-details.component.ts

```diff
@@ -253,4 +253,10 @@ export class ExperimentDetailsComponent implements OnInit, OnDestroy {
       return StatusEnum.CREATED;
     }
   }
+
+  private showGraphF(response: string): boolean {
```

Comment:
Nit, let's use a slightly more descriptive name here. Should this be `shoGraphFn` instead?

Reply:
Yes, sure! I fixed it! Let me know if it works for you!

Reply:
Looks good, thanks!